css/properties/-webkit-nbsp-mode.json

```
@@ -20,7 +21,7 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": "≤13.1"
+              "version_added": "≤5.1"
```

See: https://github.com/caugner/css-properties/blame/safari/properties.css

```suggestion
              "version_added": "≤4"
```